src/OpenFeature.Contrib.Providers.GOFeatureFlag/GoFeatureFlagProvider.cs

```diff
@@ -87,6 +91,27 @@ private void InitializeProvider(GoFeatureFlagProviderOptions options)
             if (options.ApiKey != null)
                 _httpClient.DefaultRequestHeaders.Authorization =
                     new AuthenticationHeaderValue("Bearer", options.ApiKey);
+
+
+            var cacheMaxSize = options.CacheMaxSize ?? 10000;
+            var cacheMaxTTL = options.CacheMaxTTL ?? TimeSpan.FromSeconds(60);
```

Comment:
Based on the code suggestions in the `Options` file, this assignments don't make sense.